wire-grpc-client/src/main/java/com/squareup/wire/GrpcMessageSink.kt

```diff
@@ -17,14 +17,13 @@ package com.squareup.wire
 
 import okio.Buffer
 import okio.BufferedSink
-import java.io.Closeable
 
 /** Writes a sequence of GRPC messages as an HTTP/2 stream. */
-class GrpcWriter<T> private constructor(
+internal class GrpcMessageSink<T> private constructor(
   private val sink: BufferedSink,
   private val messageAdapter: ProtoAdapter<T>,
   private val grpcEncoding: String
-) : Closeable by sink {
+) : MessageSink<T> {
```

Comment:
You should still be able to do `MessageSink<T>, Closeable by sink` and omit the explicit function below. Same for reader